src/BenchmarkDotNet/ConsoleArguments/CommandLineOptions.cs

```diff
@@ -190,6 +190,12 @@ public class CommandLineOptions
         [Option("AOTCompilerMode", Required = false, Default = MonoAotCompilerMode.mini, HelpText = "Mono AOT compiler mode, either 'mini' or 'llvm'")]
         public MonoAotCompilerMode AOTCompilerMode { get; set; }
 
+        [Option("wasmAot", Required = false, Default = false, HelpText = "When using the wasm runtime, BenchmarkDotNet ahead-of-time compile the benchmark assemblies?")]
+        public bool WasmAot { get; set; }
```

Comment:
could you extend the `MonoAotCompilerMode` enum with `wasm` value and use the `AOTCompilerMode` property instead of adding a new one?